oncopipe/oncopipe/__version__.py

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
Were these modifications intentionally included in this PR?

Reply:
Yeah, this is the solution I came up with to fix the broken relative_symlink function, which was approved by Bruno in a previous PR. This retains the original relative_symlink functionality for backwards compatibility, but adds an option for making within-module symlinks truly relative. Hopefully a best-of-both-worlds solution.

Reply:
Also I forgot to increase the version when I changed oncopipe previously, so this fixes that.